tests/api/test_narrowcast_message.py

```diff
@@ -92,6 +93,67 @@ def test_narrowcast_simple_text_message(self):
             }
         )
 
+    @responses.activate
+    def test_narrowcast_recipient_simple_text_message(self):
+        responses.add(
+            responses.POST,
+            LineBotApi.DEFAULT_API_ENDPOINT + '/v2/bot/message/narrowcast',
+            json={}, status=200,
+            headers={'X-Line-Request-Id': '5b59509c-c57b-11e9-aa8c-2a2ae2dbcce4'},
```

Comment:
[very nits]
This request id is used in `RedeliveryRecipient` below.
Although this is test code, I think that it is better to change the value of the response's `X-Line-Request-Id` to prevent confusion.

Reply:
ok, fixed 👍